src/util/atoms.ts

```diff
@@ -61,3 +61,10 @@ export const walletInfoAtom = atomWithStorage<{
     musig: null,
   },
 });
+
+export const themeAtom = atomWithStorage<"light" | "dark">(
+  "themeV1",
+  globalThis.window && window.matchMedia("(prefers-color-scheme: dark)").matches
+    ? "dark"
+    : "light",
```

Comment:
Do we want to default to light mode if no preferences? Given the mocks were mostly for dark mode.

Reply:
good question, not sure of the answer but will probably be answered down the process